etc/dbus-serialbattery/dbus-serialbattery.py

```diff
@@ -170,11 +170,16 @@ def get_port() -> str:
     # else the error throw a lot of timeouts
     sleep(16)
 
-    if port.endswith("_Ble") and len(sys.argv) > 2:
+    if port.endswith("_Ble"):
         """
         Import ble classes only, if it's a ble port, else the driver won't start due to missing python modules
         This prevent problems when using the driver only with a serial connection
         """
+
+        if len(sys.argv) <= 2
+            logger.error("missing bluetooth address")
```

Comment:
Why running the rest of the code, if a important part is missing and therefore the driver could not work?

Reply:
No reason, the reason I added that line is cause I want a proper error message; a message that informs the user what has gone wrong. Instead of silence as it was before.

We can add an else statement to stop the rest from running.

Reply:
Fixed.